ios/MListenerGlobal.m

```diff
@@ -0,0 +1,33 @@
+#import "MListenerGlobal.h"
+
+@implementation MListenerGlobal
+
+@synthesize mListener;
+
+static NSMutableDictionary *mListenerDictionary = nil;
+
++ (id)getListener:(NSString *)accountName accountNumber:(NSString *)accountNumber {
+    if (!mListenerDictionary) {
+        mListenerDictionary = [[NSMutableDictionary alloc] initWithCapacity:20];
```

Comment:
Do you know why specifically capacity is set to 20? @matejdr

Reply:
is this us or the library itself?

Reply:
@simonso-nzme it's me, we can survive with 1. This stores a list of listeners with different account details. So I picked 20 as possible combination of "customerId" and "market" combinations. But realistically we only need one. Should I change to less or remove all together?

Reply:
No, it's fine, if it works as it is.
Haven't touched objective C for a while, I thought NSMutableDictionary is mutable? so you don't really need to assign a capacity. just keep adding and it will mutate as it goes.

Reply:
Actually, I had a read, that is just initial capacity, if it needs more it will mutate, so 20 is fine.

Reply:
yeah, read about it as well yesterday and it only reserves memory, while adding more will add more memory on the fly. just thought it was faster this way, but can remove if you want.

Reply:
nah is fine, no harm. is merged :)